src/components/PlayerProfile.vue

```diff
@@ -162,17 +165,19 @@ export default {
     $route: 'executeQuery',
   },
   methods: {
+    fromNow(date) {
+      return `About ${countdown(new Date(), date, countdown.ALL, 1)} ago`;
+    },
     formatDate(date) {
       if (date == null) {
         return '-';
       }
       const d = new Date(date);
-      return (d.getFullYear() === 1969) ? 'Never' : d.toLocaleString();
+      return (d.getFullYear() < 2000) ? 'Never' : d.toLocaleString();
```

Comment:
Does the API ever return such an invalid date? If not, we can probably remove this code and rely on the `null` check above.
If the API actually returns `0` for some cases you can change the check above to `if (!date)`